core/errors.go

```diff
@@ -29,6 +30,8 @@ func isTemporary(info httperror.Info, err error) bool {
 		return true
 	case "CH001": // request timed out
 		return true
+	case "CH012": // conflict
+		return true
```

Comment:
I think we need something like the c++ library's [MessageDifferencer](https://github.com/google/protobuf/blob/master/src/google/protobuf/util/message_differencer.h)